docs/Preferences.md

```diff
@@ -19,7 +19,7 @@
 
   Can be accessed through `xml.server.vmargs`.
 
-  Setting up proxy:
+  eg. Setting up proxy:
```

Comment:
Could we make this a header instead? Then it would look something like:

Can be accessed through `xml.server.vmargs`.


### Setting up proxy:

A proxy can be defined in the following way if there is a system proxy: